pkg/crypto/tinkcrypto/crypto.go

```diff
@@ -46,6 +60,11 @@ func (t *Crypto) Encrypt(msg, aad []byte, kh interface{}) ([]byte, []byte, error
 		return nil, nil, errBadKeyHandleFormat
 	}
 
+	ps, err := keyHandle.Primitives()
+	if err != nil {
+		return nil, nil, fmt.Errorf("get primitives: %w", err)
```

Comment:
Should this be `failed to get primitives` instead of just `get primitives`?

Reply:
not really.. unless we add the function name .. I didn't added it because it was not added elsewhere in the function